src/tools/perf/api/libperf.h

```diff
@@ -217,11 +216,12 @@ typedef struct ucx_perf_params {
     } uct;
 
     struct {
-        unsigned               nonblocking_mode; /* TBD */
-        ucp_perf_datatype_t    send_datatype;
-        ucp_perf_datatype_t    recv_datatype;
-        size_t                 am_hdr_size; /* UCP Active Message header size
-                                               (not included in message size) */
+        ucp_perf_datatype_t     send_datatype;
+        ucp_perf_datatype_t     recv_datatype;
+        size_t                  am_hdr_size; /* UCP Active Message header size
+                                                (not included in message size) */
+        struct sockaddr_storage dmn_local_addr;
+        struct sockaddr_storage dmn_remote_addr;
```

Comment:
WDYT, maybe it would be better to add both `dmn_local/remote_addr` fields in the separate PR and make this PR contain `perftest_daemon` changes only. I mean that in the current patch we only introduce and define them to default value, so, from my point of view, it makes sense to introduce them in the patch, where we would have some logic connected to these variables.

If you disagree with the point described above, pls consider adding comment that says that these variables aren't used for now.

Reply:
I would keep them in this PR for 2 reasons:
1. In this PR we introduce new command line arguments and parser exactly for these fields (`--daemon-local` and `--daemon-remote`). So it's not just for assigning default values)
IMO these fields are actually "used" for parsing, but not for communication yet
2. If I remove these addresses and related CLI and parsing, then the PR would be around 200 LOC. Committing in such a small chunks comes with a lot of overhead, so I would prefer to keep a reasonable PR size. So that overall we have ~3 commits for this feature

As for the comment - well, if you have strong opinion I can add it, but I hardly see the benefit.
I mean I will have to remember to remove it in the subsequent commit..
Please let me know

Reply:
Oh, sorry I missed that part, my bad :)